maven-wrapper-distribution/src/resources/mvnw

```diff
@@ -160,7 +160,7 @@ find_maven_basedir() {
 # concatenates all lines of a file
 concat_lines() {
   if [ -f "$1" ]; then
-    echo "$(tr -s '\n' ' ' < "$1")"
+    echo "$(tr -s '\r\n' ' ' < "$1")"
```

Comment:
Needs to be  rebased on top of https://github.com/apache/maven-wrapper/pull/79 later (removed that unnecessary echo call).

Reply:
I've updated the PR :)